plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -863,6 +1013,8 @@ writeBits (BuiltinByteString bs) (BuiltinList ixes) (BuiltinBool bit) =
     BuiltinSuccessWithLogs logs bs' -> traceAll logs $ BuiltinByteString bs'
 {-# OPAQUE writeBits #-}
 
+-- | Creates a bytestring of a given length by repeating the given byte.
+-- Fails if the byte, second argument, is not in range @[0,255]@ or the length is negative.
 replicateByte ::
```

Comment:
Also [fails if the length is greater than 8192](https://github.com/IntersectMBO/plutus/blob/c86aca85c4104114b680108640ab6e2bf5c00f2c/plutus-core/plutus-core/src/PlutusCore/Bitwise.hs#L49).